conftest.py

```diff
@@ -8,14 +8,19 @@
 from fxapom.fxapom import DEV_URL, FxATestAccount, PROD_URL
 
 
-def pytest_funcarg__mozwebqa(request):
+@pytest.fixture(scope='session')
+def capabilities(capabilities):
+    capabilities.setdefault('tags', []).append('oneanddone')
+    return capabilities
+
+
+@pytest.fixture
+def click_login(base_url, selenium):
     fxa_login_button_locator_css = 'button.signin'
-    mozwebqa = request.getfuncargvalue('mozwebqa')
-    mozwebqa.selenium.get('%s/' % mozwebqa.base_url)
-    WebDriverWait(mozwebqa.selenium, mozwebqa.timeout).until(
+    selenium.get('%s/' % base_url)
+    WebDriverWait(selenium, 10).until(
```

Comment:
A later refactor could remove this hard-coded timeout, but let's leave it for now.